pkg/app/piped/executor/registry/registry.go

```diff
@@ -104,4 +105,5 @@ func init() {
 	ecs.Register(defaultRegistry)
 	wait.Register(defaultRegistry)
 	waitapproval.Register(defaultRegistry)
+	custom.Register(defaultRegistry)
```

Comment:
```suggestion
	customsync.Register(defaultRegistry)
```
should rename it like this